int/api/webcreator.go

```diff
@@ -45,12 +45,35 @@ func PrepareForWebsiteHandler(params operations.WebsiteCreatorPrepareParams) mid
 				})
 	}
 
+	archive, err := ioutil.ReadAll(params.Zipfile)
```

Comment:
Change `ioutil`  by `io` or `os` as it's described here:  
[https://pkg.go.dev/io/ioutil](https://pkg.go.dev/io/ioutil)

Reply:
c'est fait